config/locales/en.yml

```diff
@@ -195,6 +195,9 @@ en:
       beds:
         non_positive: "Number of bedrooms has to be greater than 0"
         over_max: "Number of bedrooms cannot be more than 12"
+        bedsits_have_max_one_bedroom: "Bedsit bedroom maximum 1"
+      proptype:
+        bedsits_have_max_one_bedroom: "Bedsit maximum 1 bedroom"
```

Comment:
Can we improve on these error messages? Neither are sentences!

Reply:
@jamesrose I agree I'm not a fan of these - I originally wrote something more verbose, but then changed them to what they are now as I was told Beth had written them so I should use them. Do you think I should query these with her?